optuna/samplers/_brute_force.py

```diff
@@ -54,7 +54,7 @@ def set_leaf(self) -> None:
         self.expand(None, [])
 
     def add_path(
-        self, params_and_search_spaces: Iterable[Tuple[str, Iterable[Any], Any]]
+        self, params_and_search_spaces: Iterable[Tuple[str, Iterable[float], Any]]
```

Comment:
```suggestion
        self, params_and_search_spaces: Iterable[Tuple[str, Iterable[float], float]]
```
The value is also in internal representation.

Reply:
Thank you for the suggestion @contramundum53 , it has been corrected in my last commit :+1: